android/demo/README.md

```diff
@@ -9,11 +9,10 @@ It's been tested on Andriod Studio Chipmunk(2021.2.1) and Android Studio Giraffe
 Assuming you have read the [requirements on the root contributing guide](https://github.com/player-ui/player/blob/main/CONTRIBUTING.md).
 
 1. Once you have Android Studio installed, you will need to go to tools->SDK Manager->SDK Platforms.
-   1.  Make sure you have **only** the following SDK installed: Android API 32.
-   *If you are using Android Giraffe, you may need to click on show package details and it will be under Android12L. (Android SDK Platrform 32)*
+   1.  Make sure you have **only** the following SDK installed: Android API 33.
 2. The next step will be to make sure you have the right (and _only_ the right) SDK Build tools and NDK. Click on the SDK Tools tab and make sure you have only the following clicked:
    1. 30.0.3
-   2. 21.4.7075529
+   2. 27.1.12297006
```

Comment:
Do we need to update our build image to have this preinstalled?

Reply:
This change wasn't actually made with this PR, this was done a while ago. It looks like our image uses 23 which still works, but I can put up a PR for that